function testcase ( ) { var foo = function ( ) ; try { foo . = 41 ; return false ; } catch ( e ) { return e instanceof TypeError ; } } runTestCase ( testcase ) ;